Regen bindings, warning fix

Manishearth opened the pull request:
The warning only happens on the latest stable (which we use in the incubator), so there's an `#[allow()]` there for now as well.

Don't signal r+ to bors, I'll do that myself when the incubator side lands.

r? @bholley

<!-- Reviewable:start -->

---

This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/servo/servo/13723)

<!-- Reviewable:end -->

Manishearth committed:
Resync bindings

Manishearth committed:
Use unsafe block on access of column count constant

highfive commented:
Heads up! This PR modifies the following files:
 * @bholley: components/style/properties/gecko.mako.rs, components/style/gecko_bindings/structs_debug.rs, components/style/gecko_bindings/structs_release.rs

highfive commented:
<img src="http://www.joshmatthews.net/warning.svg" alt="warning" height=20> **Warning** <img src="http://www.joshmatthews.net/warning.svg" alt="warning" height=20>

* These commits modify **unsafe code**. Please review it carefully!
* These commits modify style code, but no tests are modified. Please consider adding a test!

Manishearth committed:
Remove test functions from nsstring_vendor

These tests are run on the real nsstring anyway. They cause linux builds to
fail since the test functions are defined twice.

bholley commented:
r=me

KiChjang commented:
@bors-servo r=bholley

bors-servo commented:
:pushpin: Commit 647f8e9 has been approved by `bholley`

<!-- @bors-servo r=bholley 647f8e9c5d2faf1127aedf580686d64746a65131 -->

KiChjang commented:
@bors-servo r-

Manishearth commented:
@bors-servo r=bholley

bors-servo commented:
:pushpin: Commit 647f8e9 has been approved by `bholley`

<!-- @bors-servo r=bholley 647f8e9c5d2faf1127aedf580686d64746a65131 -->

bors-servo commented:
:hourglass: Testing commit 647f8e9 with merge d9bb663...

bors-servo commented:
:sunny: Test successful - [arm32](http://build.servo.org/builders/arm32/builds/3503), [arm64](http://build.servo.org/builders/arm64/builds/3515), [linux-dev](http://build.servo.org/builders/linux-dev/builds/3584), [linux-rel-css](http://build.servo.org/builders/linux-rel-css/builds/210), [linux-rel-wpt](http://build.servo.org/builders/linux-rel-wpt/builds/210), [mac-dev-unit](http://build.servo.org/builders/mac-dev-unit/builds/3580), [mac-rel-css](http://build.servo.org/builders/mac-rel-css/builds/3560), [mac-rel-wpt1](http://build.servo.org/builders/mac-rel-wpt1/builds/261), [mac-rel-wpt2](http://build.servo.org/builders/mac-rel-wpt2/builds/262), [windows-dev](http://build.servo.org/builders/windows-dev/builds/1468)
